src/Stratis.Bitcoin/Consensus/ConsensusExceptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Consensus
+{
+    public class ConsensusException : Exception
+    {
+        public ConsensusException() : base()
```

Comment:
do you really want a consensus exception with no error message?

Reply:
You are probably right, I will add a message.